lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,19 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color, even if the output is not a TTY') do |_o|
```

Comment:
I just realized that these options are wider than the default width, causing the help text to layout sub-optimally:

```
    -b, --backtrace                  Enable full backtrace.
    -c, --color, --colour            Deprecated: enable color in the output.
        --force-color, --force-colour
                                     Force the output to be in color, even if the output is not a TTY
        --no-color, --no-colour      Force the output to not be in color, even if the output is a TTY
    -p, --[no-]profile [COUNT]       Enable profiling of examples and list the slowest examples (default: 10).
    -w, --warnings                   Enable ruby warnings
```

Notice that the description for these is on the next line so it isn't smashed together with the options themselves.  If you set `parser.summary_width = 34` then it lays out nicely:

```
    -b, --backtrace                    Enable full backtrace.
    -c, --color, --colour              Deprecated: enable color in the output.
        --force-color, --force-colour  Force the output to be in color, even if the output is not a TTY
        --no-color, --no-colour        Force the output to not be in color, even if the output is a TTY
    -p, --[no-]profile [COUNT]         Enable profiling of examples and list the slowest examples (default: 10).
    -w, --warnings                     Enable ruby warnings
```

Mind doing that?